portlayer/network/scope.go

```diff
@@ -64,6 +65,10 @@ func (s *Scope) IPAM() *IPAM {
 	return s.ipam
 }
 
+func (s *Scope) NetworkName() string {
```

Comment:
Unless NetworkName is something that needs to be computed during runtime, just make `networkName` public and avoid the getter/setter pattern.